lib/db-manager.js

```diff
@@ -377,14 +377,14 @@ var dbManager = module.exports = {
    /**
     * Returns a promise which resolves to a Status object.
     */
-   getCommitStatus: function(repo, sha) {
+   getCommitStatuses: function(repo, sha) {
       var q_select = '\
          SELECT * FROM commit_statuses \
          WHERE commit = ? AND repo = ?';
 
       return db.query(q_select, [sha, repo]).then(function(rows) {
-         if (rows.length === 1) {
-            return Status.getFromDB(rows[0]);
+         if (rows.length > 0) {
+            return rows.map(row => Status.getFromDB(row));
```

Comment:
Should this perhaps return an empty array instead of null when now rows are found? 

Looks like it's eventually ORed to [an empty array anyway](https://github.com/iFixit/pulldasher/blob/556979983ed2eca064ca90c6ce104caf122754b8/models/pull.js#L14) so I think it makes sense to do that here so the type is consistent.

Reply:
e3e75488efa9428d121047d16b2a645ac5f8c639